backend/src/xfd_django/xfd_api/views.py

```diff
@@ -658,6 +687,510 @@ async def invoke_scheduler(current_user: User = Depends(get_current_active_user)
     return response
 
 
+@api_router.get(
+    "/services/",
+    tags=["Retrieve Stats"],
+)
+async def get_services(
```

Comment:
See other endpoint definitions. We want these as minimal as possible so move to a function in api_methods/